mods/cnc/fluent/rules.ftl

```diff
@@ -528,10 +528,11 @@ actor-atwr =
     .name = Advanced Guard Tower
     .description =
     All-purpose defensive structure.
+    Requires power to operate.
       Strong vs Aircraft and Infantry.
       Weak vs Tanks.
     .encyclopedia =
-    Advanced defense for GDI that fires volleys of high explosive missiles at both ground and air targets. Effective versus everything.
+    Advanced defense for GDI that fires volleys of high explosive missiles at both ground and air targets.
```

Comment:
The effective vs everything gives quite a bit of style. Maybe reword instead of removing?